src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayFormat.cs

```diff
@@ -268,7 +268,7 @@ public class SymbolDisplayFormat
                 globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.OmittedAsContaining,
                 typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces,
                 genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters,
-                miscellaneousOptions: SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers,
+                miscellaneousOptions: SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers | SymbolDisplayMiscellaneousOptions.ExpandValueTuple,
```

Comment:
> | SymbolDisplayMiscellaneousOptions.ExpandValueTuple [](http://example.com/codeflow?start=96&length=53)

Why is this change necessary?

Reply:
To avoid a cycle when completing tuple members and some member has tuple in signature (e.g., `bool System.IEquatable<(T1, T2, T3, T4, T5, T6, T7, TRest)>.Equals((T1, T2, T3, T4, T5, T6, T7, TRest) other)` in tests), it will go through `SymbolDisplayVisitor` to compute the explicitly implemented member name which will in turn bind the tuple type and continue the cycle.

Reply:
What is the source of the cycle in this case?

Reply:
If you mean how the cycle happens, I tried to describe that in the comment above, but here it is in more detail:
1. compilation is forced to complete
2. tuple members are being created
3. explicitly implemented members are creating their name
4. they use `SymbolDisplayVisitor` for the interface type
5. the interface type contains the tuple, so the visitor queries the tuple's members to display it
6. go to step 2